### PR TITLE
chore(curriculum): remove `__locals` from shortest path project

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Your dictionary should have an `A` key.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     "A" in graph
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md:19]:
```suggestion
    "A" in my_graph
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Your `A` key should have `B` as the value.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     graph["A"] == "B"
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md:29]:
```suggestion
    my_graph["A"] == "B"
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Your dictionary should have an `B` key.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     "B" in graph
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md:39]:
```suggestion
    "B" in my_graph
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md [hunk 4]
```diff
@@ -45,7 +45,7 @@ Your `B` key should have `A` as the value.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     graph["B"] == "A"
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md:49]:
```suggestion
    my_graph["B"] == "A"
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md [hunk 5]
```diff
@@ -55,7 +55,7 @@ Your dictionary should have two keys.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     len(graph) == 2
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557709b0aee699a6a00528c.md:59]:
```suggestion
    len(my_graph) == 2
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ Your dictionary should have 3 keys — `A`, `B`, and `C`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     key_list = ["A", "B", "C"]
     len(graph) == 3 and all(key in graph for key in key_list)
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md:22]:
```suggestion
    key_list = ["A", "B", "C"]
    len(my_graph) == 3 and all(key in my_graph for key in key_list)
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md [hunk 7]
```diff
@@ -28,7 +28,7 @@ Your dictionary should have 3 keys — `A`, `B`, and `C`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     graph["A"] == "B"
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md:32]:
```suggestion
    my_graph["A"] == "B"
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md [hunk 8]
```diff
@@ -38,7 +38,7 @@ Your dictionary should have 3 keys — `A`, `B`, and `C`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     type(graph["B"]) is list
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md:42]:
```suggestion
    type(my_graph["B"]) is list
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md [hunk 9]
```diff
@@ -48,7 +48,7 @@ The value of `my_graph["B"]` should be a list containing the other two nodes.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     len(graph["B"]) == 2 and "A" in graph["B"] and "C" in graph["B"]
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md:52]:
```suggestion
    len(my_graph["B"]) == 2 and "A" in my_graph["B"] and "C" in my_graph["B"]
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md [hunk 10]
```diff
@@ -58,7 +58,7 @@ The value of `my_graph["C"]` should be the connected node.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     graph["C"] == "B"
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557712d77ce2d9bd7e63afd.md:62]:
```suggestion
    my_graph["C"] == "B"
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 11]
```diff
@@ -15,7 +15,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     key_list = ["A", "B", "C", "D"]
     len(graph) == 4 and all(key in graph for key in key_list)
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:20]:
```suggestion
    key_list = ["A", "B", "C", "D"]
    len(my_graph) == 4 and all(key in my_graph for key in key_list)
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 12]
```diff
@@ -26,7 +26,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     type(graph["A"]) is list
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:30]:
```suggestion
    type(my_graph["A"]) is list
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 13]
```diff
@@ -36,7 +36,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     len(graph["A"]) == 2 and "B" in graph["A"] and "D" in graph["A"]
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:40]:
```suggestion
    len(my_graph["A"]) == 2 and "B" in my_graph["A"] and "D" in my_graph["A"]
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 14]
```diff
@@ -46,7 +46,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     type(graph["B"]) is list
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:50]:
```suggestion
    type(my_graph["B"]) is list
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 15]
```diff
@@ -56,7 +56,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     len(graph["B"]) == 2 and "A" in graph["B"] and "C" in graph["B"]
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:60]:
```suggestion
    len(my_graph["B"]) == 2 and "A" in my_graph["B"] and "C" in my_graph["B"]
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 16]
```diff
@@ -66,7 +66,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     type(graph["C"]) is list
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:70]:
```suggestion
    type(my_graph["C"]) is list
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 17]
```diff
@@ -76,7 +76,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     len(graph["C"]) == 2 and "B" in graph["C"] and "D" in graph["C"]
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:80]:
```suggestion
    len(my_graph["C"]) == 2 and "B" in my_graph["C"] and "D" in my_graph["C"]
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 18]
```diff
@@ -86,7 +86,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     type(graph["D"]) is list
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:90]:
```suggestion
    type(my_graph["D"]) is list
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md [hunk 19]
```diff
@@ -96,7 +96,7 @@ Your dictionary should have 4 keys called `A`, `B`, `C`, and `D`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     len(graph["D"]) == 2 and "A" in graph["D"] and "C" in graph["D"]
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557716aadbd2d9c42c0e69a.md:100]:
```suggestion
    len(my_graph["D"]) == 2 and "A" in my_graph["D"] and "C" in my_graph["D"]
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557924d47c325bf27afbe51.md [hunk 20]
```diff
@@ -24,7 +24,7 @@ The `target` parameter should have the default value of an empty string.
 ```js
 ({ test: () => assert(__pyodide.runPython(`
     import inspect
-    foo = __locals.get("shortest_path")
+    foo = shortest_path
     sig = str(inspect.signature(foo))
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6557924d47c325bf27afbe51.md:28]:
```suggestion
    sig = str(inspect.signature(shortest_path))
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6566195b0a021bb660b2b4b1.md [hunk 21]
```diff
@@ -15,7 +15,7 @@ Now modify `my_graph["B"]` into a list of tuples. The `B-C` distance is `4`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     type(graph["B"]) is list and all(type(i) is tuple for i in graph["B"])
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6566195b0a021bb660b2b4b1.md:19]:
```suggestion
    type(my_graph["B"]) is list and all(type(i) is tuple for i in my_graph["B"])
```

File: curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6566195b0a021bb660b2b4b1.md [hunk 22]
```diff
@@ -25,7 +25,7 @@ Now modify `my_graph["B"]` into a list of tuples. The `B-C` distance is `4`.
 
 ```js
 ({ test: () => assert(__pyodide.runPython(`
-    graph = __locals.get("my_graph")
+    graph = my_graph
     tuples = [("A", 3), ("C", 4)]
     len(graph["B"]) == 2 and all(t in graph["B"] for t in tuples)
```
Comment (Dario-DC) [inline curriculum/challenges/english/07-scientific-computing-with-python/learn-algorithm-design-by-building-a-shortest-path-algorithm/6566195b0a021bb660b2b4b1.md:30]:
```suggestion
    tuples = [("A", 3), ("C", 4)]
    len(my_graph["B"]) == 2 and all(t in my_graph["B"] for t in tuples)
```

**Review by Dario-DC — CHANGES_REQUESTED** [review]

**Review by naomi-lgbt — APPROVED** [review]
No locals found in block

**Review by Dario-DC — APPROVED** [review]

**Pull request merged**